sycl/test/regression/private_array_init_test.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Does it make sense to run the test on anything except CPU ?

Reply:
I don't know about GPU, but for accelerator it makes sense because the issue was detected on FPGA emulator initially.

Reply:
What about HOST?

Reply:
Initially the issue was detected on the step of reverse translation of code from SPIR-V to IR.
There is already added test in SPIR-V translator. However, @bader said that it would be nice to have common test that checks that this option supported by SYCL properly. So, I would suggested to leave it as regression check for all device types.

Reply:
Ok, I don't think it makes any sense to run it on host though.

Reply:
Ok, removed run command on host as useless.